### DISCUSSION TITLE
Private dotfiles repo support

### DISCUSSION THREAD

**Discussion by sheldonhull** [community/community#7837] [Codespaces] [answered]
I'd like to use a private repo for running my dotfiles.

This isn't currently supported.
Since I've already allowed codespaces to access all my own repos, I'd expect this permission to allow the bootstrapping process to reach out and clone the private dotfiles repo I've specified in my user settings.

Any issue/item I can follow for this, or is there any work-around other than perhaps embedding a token in my dotfiles url in settings?

**Comment by asciimike** [comment]
Support for selecting a dotfiles repos will be supported in the near future; I can update this when it ships.

**Comment by asciimike** [comment, marked as answer]
We have shipped this feature: https://github.blog/changelog/2021-12-01-codespaces-can-install-dotfiles-from-any-repo
